### PR TITLE
✨ New Page - Company History 

### LINKED ISSUE #1074: ✨ New Page - Company | History
Migrate this page https://www.ssw.com.au/ssw/Company/History/

New URL should be /company/history

There are other pages that will use the same template, so keep that in mind when building it.

This history timline should be a seperate component

```[tasklist]
### Tasks
- [x] Build the new template
- [x] Build the timeline component, should be editable in Tina
- [x] Add redirect in FrontDoor
- [x] Update megamenu link
```

### PR DESCRIPTION
**Pull request by jackreimers**
Migrates the old company history page to the new website.
https://www.ssw.com.au/ssw/Company/History/

Fixes #1074 

Affected routes:

- `/company/history`

image
**Figure: The migrated company history page**

### CODE REVIEW

File: .tina/collections/company.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  label: "Company Pages",
+  name: "company",
+  format: "mdx",
+  path: "content/company",
+  ui: {
+    router: ({ document }) => {
+      return `/company/${document._sys.filename}`;
+    },
+  },
+  fields: [
```
Comment (bradystroud) [inline .tina/collections/company.tsx:16]: Add a switch for breadcrumb

File: components/company/historyTimelineCard.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  const flagUrl = sswCountries.find((f) => f.label === location)?.flagUrl;
+
+  return (
+    <div className="flex gap-3 even:flex-row-reverse">
+      <div className="flex-1 p-6"></div>
+      <div className="inline-flex h-full flex-initial flex-col rounded-2xl bg-sswRed px-12 text-3xl text-white shadow">
+        <div className="flex-1"></div>
+        <p className="font-bold">{year}</p>
+        <div className="flex-1"></div>
+      </div>
```
Comment (bradystroud) [inline components/company/historyTimelineCard.tsx]: Try a solution without a div sandwich 🥪

**Review by jaydenalchin — CHANGES_REQUESTED** [review]
Love the squircles we're moving toward but:

1. Please remove the little drop shadow from these cards 
2. Can we make the cards in each row connect as they did in the old page? 

E.G.
image
**Figure: Only 2 rounded corners on each card**

File: pages/company/[filename].tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      title: m.title,
+      location: m.location as HistoryTimelineCardProps["location"],
+      description: m.description,
+    })) || [];
+
+  return (
+    <div>
+      <RecaptchaContext.Provider
+        value={{ recaptchaKey: props.env.GOOGLE_RECAPTCHA_SITE_KEY }}
+      >
```
Comment (bradystroud) [inline pages/company/[filename].tsx]: Remove this - we are not using it

File: pages/company/[filename].tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+export const getStaticProps = async ({ params }) => {
+  const tinaProps = await client.queries.companyContentQuery({
+    relativePath: `${params.filename}.mdx`,
+  });
+
+  const canonical = `${tinaProps.data.global.header.url}company/${params.filename}`;
+  const seo = tinaProps.data.company.seo;
+  if (seo) {
+    seo.canonical = canonical;
+  }
```
Comment (bradystroud) [inline pages/company/[filename].tsx]: Understand what this does, do we need it?

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by jaydenalchin — APPROVED** [review]

File: pages/company/[filename].tsx (hunk 4 above)
Comment (jackreimers) [reply pages/company/[filename].tsx]: @bradystroud 

From [this article](https://moz.com/learn/seo/canonicalization), a canonical url prevents crawlers indexing a site multiple times when the url is slightly different, ie http:// => https:// or /consulting/cms => /consulting/cms?r... 

This piece of code is setting that canonical url on each page, which we can see in the head tag of each page:

image
**Figure: The canonical url element in the head tag of `/consulting/cms`**

We should probably make a task to check each template to ensure they have one. It looks like some such as the [training template](https://www.ssw.com.au/events/angular-superpowers-tour) are missing it.

File: components/company/historyTimelineCard.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            <span className="mx-2 inline-block">
+              <Image
+                src={flagUrl}
+                alt={"The flag of " + location}
+                height={10}
+                width={24}
+              />
+            </span>
+          ) : (
+            <></>
```
Comment (amankumarrr) [inline components/company/historyTimelineCard.tsx]: can we do something 

{location && (<span className="mx-2 inline-block">
              <Image
                src={flagUrl}
                alt={"The flag of " + location}
                height={10}
                width={24}
              />
            </span>)

**Review by amankumarrr — APPROVED** [review]
LGTM

**Review by bradystroud — APPROVED** [review]

**Pull request merged**